.codecov.yml

```diff
@@ -0,0 +1,3 @@
+codecov:
+  branch: master
+  # strict_yaml_branch: master # Enable this if we want to use the yml file in master to dictate the reports for all branches
```

Comment:
Maybe a good idea to uncomment this so that the yaml file is restricted to master.